vulnerable_people_form/integrations/postcode_lookup_helper.py

```diff
@@ -111,10 +163,13 @@ def get_addresses_from_postcode(postcode):
                     )
             return values
     elif response.status_code == HTTPStatus.UNAUTHORIZED.value:
-        logger.error(_create_postcode_lookup_failure_log_message("Unauthorised request submitted to API - Invalid ORDNANCE_SURVEY_PLACES_API_KEY", postcode, response.text))  # noqa
+        logger.error(_create_postcode_lookup_failure_log_message(
+            "Unauthorised request submitted to API - Invalid ORDNANCE_SURVEY_PLACES_API_KEY", postcode,
+            response.text))  # noqa
```

Comment:
I think you can remove the # noqa identifiers as you have made them nicer. https://stackoverflow.com/questions/45346575/what-does-noqa-mean-in-python-comments

I'll have a look at the rest of the code too.